src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertListToObjects(List<String> list) {
```

Comment:
The method name `convertListToObjects` is not very descriptive of what type of objects it converts to. Consider renaming the method to be more specific, such as `convertToTransactions` or `parseTransactions`.